sys/include/net/netopt.h

```diff
@@ -202,6 +202,13 @@ typedef enum {
     NETOPT_CCA_THRESHOLD,
 
     /**
+     * @brief CCA mode for the radio transceiver
+     */
+    NETOPT_CCA_MODE,            /**< get/set the CCA mode as uint8_t
+                                 *   corresponding to the respective PHY standard,
+                                 *   e.g Std 802.15.4 */
```

Comment:
Just something along the like

```C
    /** 
     * @brief CCA mode for the radio transceiver
     *
     * The type of this option depends on the PHY standard used.
     *  - IEEE 802.15.4: @ref netdev2_ieee802154_ccc_mode_t
     */
    NETOPT_CCA_MODE,
```

Further PHY protocols can add to this later on.